node/src/main/java/io/horizen/eon/forks/F7Fork.java

```diff
@@ -0,0 +1,51 @@
+package io.horizen.eon.forks;
+
+import io.horizen.account.fork.Version1_5_0Fork;
+import io.horizen.fork.OptionalSidechainFork;
+import io.horizen.fork.SidechainForkConsensusEpoch;
+import io.horizen.utils.Pair;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * EON fork 7 (introduced in Version 1.5.0)
+ * Disables the backward transfers submission as part of the strategy which stops any cross-chain transfer operation between Mainchain and Sidechain with a view to EON2 migration.
+ */
+public class F7Fork extends EONFork {
```

Comment:
Shouldn't be added to EonForkConfigurator class?

Reply:
Fixed